src/infra/dataverse/sparql/sparqlGateway.ts

```diff
@@ -29,7 +29,9 @@ export const sparqlGateway: DataversePort = {
       flow(A.map(buildStrExpression), a => a.join(' || '))(filters)
 
     const byPropertyFilter = (filter: ByPropertyQueryFilter): string =>
-      `contains(lcase(str(?${filter?.property})), "${filter?.value.toLowerCase()}" )`
+      `contains(lcase(str(?${filter?.property})), "${filter?.value
+        .toLowerCase()
+        .replace(/"/g, '\\"')}" )`
```

Comment:
Not quite right... What you want is to _escape_ in order to safely include the content within a SPARQL string. For this, you have to consider all the cases, for instance the character `\`.

Because this is a more general concern to convert literal text into a format safe for inclusion in a particular context,  I'd recommand to have dedicated functions for that.

Reply:
@ccamel 
Your are completely right.
I added a simple fix because this was the only character that was not correctly handled in the `sparql` query.
I pushed a more generic function about escaping, tell me what do you think 😌